src/backend/base/langflow/components/models/groq.py

```diff
@@ -60,29 +59,6 @@ class GroqModel(LCModelComponent):
         ),
     ]
 
-    def get_models(self) -> list[str]:
```

Comment:
I don't think we should remove this. You should probably just update it to return the default list in case anything fails.

Reply:
ok!
So we will keep the update_build_config also?

Reply:
Added it as default model list and updates if model list available from API.